.travis.yml

```diff
@@ -1,17 +1,24 @@
 language: android
 jdk: oraclejdk8
 before_install:
-- nvm install 6
+- nvm install 8
 - node --version
 install:
-- cd examples && cd RNOneSignal && npm install
+- cd examples
+- cd RNOneSignal
+- npm install -g yarn
+- yarn
 android:
   components:
   - build-tools-23.0.1
+  - build-tools-26.0.2
+  - build-tools-27.0.3
   - android-23
```

Comment:
Do we need all 3 of these `android-` versions? Think we can remove `android-23`?

Reply:
You're welcome to try, but according to my tests we do need all those dependencies.